src/frontend/src/markdown-pages/about.md

```diff
@@ -0,0 +1,66 @@
+---
+path: '/about'
+title: 'About'
+---
+
+## Introduction
+
+One of the key features of Seneca's open source involvement has been the
+emphasis on sharing what we're working on, teaching, and learning through blogging.
+We believe that one of the most rewarding parts of learning to work in the
+open source community is discovering that one can become part of the fabric
+of the web, find a voice, and build a following.
+
+We also believe that reading each other's blog posts is important. In the blog
+posts of our colleagues, we find that we are not alone in our struggles to make
+things work, our interests in various topics, and that imposter syndrome isn't
+something unique to "me."
+
+To better enable the discovery of blogs within our community, we set up an open
+source blog [Planet](<https://en.wikipedia.org/wiki/Planet_(software)>): an aggregated
+feed of blog posts from Seneca faculty and students working on open source in a
+single page. Our old blog Planet used to live at http://zenit.senecac.on.ca/~chris.tyler/planet/, and was run faithfully by Chris Tyler for more than a decade. We've made a re-creation of what it looked like at [http://telescope.cdot.systems/planet](http://telescope.cdot.systems/planet) if you want to see it..
```

Comment:
So the link here is to production.  We can't swap in the current URL via metadata somehow in the Gatsby build?  It would be nice if this was `dev.telescope.cdot.systems` on staging and `telescope.cdot.systems` in production and `http://localhost:3000` locally.  Not critical, but it would be nice.

Reply:
I'll file a new issue for the swapping of URLs after this lands... :D?